charts/kube-state-metrics/Chart.yaml

```diff
@@ -6,7 +6,7 @@ keywords:
 - monitoring
 - prometheus
 - kubernetes
-version: 2.9.7
+version: 2.11.1
```

Comment:
Assuming this should be `2.10.0` if we're bumping MINOR?

Reply:
My bad - fixed :)